sample/shared/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+plugins {
+    kotlin("multiplatform")
+    id("com.android.library")
+    id("org.jetbrains.compose")
+}
+
+@OptIn(org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi::class)
+kotlin {
+    targetHierarchy.default()
+
+    androidTarget {
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = "1.8"
```

Comment:
Java Version `1.8` should not be used anymore as it's **EOL**.
Please use 11 or 17.

To change it for the whole project, edit the root `build.gradle.kts` like this https://github.com/DATL4G/KCEF/blob/master/build.gradle.kts#L32

Reply:
Thanks for your review! I have upgraded to Java 17